### ISSUE TITLE
hyrolo-org breaks org-capture

### ISSUE THREAD

**Issue by AquariusDue** [rswgnu/hyperbole#599]
Hi there, first off I really enjoy using Hyperbole, I've barely scratched the surface and I find it incredibly awesome and super useful.

The one problem I've encountered that I can't really tell if it's user error or a subtle bug is that after using hyrolo-org via "M-x", searching/greping for a string and then simply killing with "C-x k" the hyrolo buffer containing the search results I can't use org-capture anymore. 

The error that appears in the echo area is "Capture template 'b': Buffer is read-only: #<killed buffer>".
That specific template is for bookmarks and first prompts for a clipboard value to paste in and afterwards prompts for tags, only after that it stops working. While another template for notes fails immediately  after invocation with the same error basically.

Steps to reproduce:
1. {M-x org-capture} something with a template
2. {M-x hyrolo-org} searching for anything
3. {C-x k \*HyRolo*)
3. try {M-x org-capture} again

After quitting Emacs org-capture works normally again so this is really puzzling to me.

org-capture 'b' template:
```
("b" "Bookmark"
entry (file+headline "~/org/notes.org" "Bookmarks")
"* [[%^C][%?]] %^g\n:PROPERTIES:\n:CREATED: %U\n:END:\n\n"
:empty-lines 1)
```

The problem I encountered seems similar to this one posted a while back on Reddit:
[Link to /r/emacs post](https://www.reddit.com/r/emacs/comments/rgskwb/users_of_gnu_hyperbole_hyrolo_why_does_my_buffer/)

**Comment by rswgnu** [comment]
Thank you for the detailed bug report.  We will look into it.

**Comment by matsl** [comment]
@AquariusDue This has been fixed with commit 68e68cd893829f3c741c5a4df161022093a32d8e. It will be in the next release and is already available through the [elpa-devel](https://elpa.gnu.org/devel/hyperbole.html) archive. Thanks again for the report,
